src/main/java/uk/gov/hmcts/cmc/claimstore/rules/MoreTimeRequestRule.java

```diff
@@ -48,11 +36,6 @@ public void assertMoreTimeCanBeRequested(Claim claim) {
         if (claim.isMoreTimeRequested()) {
             validationErrors.add(ALREADY_REQUESTED_MORE_TIME_ERROR);
         }
-
-        if (claimDeadlineService.isPastDeadline(nowInLocalZone(), newDeadline)) {
-            validationErrors.add(PAST_DEADLINE_ERROR);
-        }
-
```

Comment:
Why are we no longer validating that the new deadline is past the deadline to extend the deadline?

Reply:
Brian said that this is how the business wants it to be.

Reply:
but that was existing feature for frontend, this change is only for caseworker

Reply:
reverted